tf-modules/ingest/sqs-message-remover.tf

```diff
@@ -0,0 +1,22 @@
+module "sqs_message_remover_lambda" {
```

Comment:
Rather than a separate TF file, could this `module` just be placed in `tf-modules/ingest/lambda-functions.tf` where the original lambda was?

Reply:
Sure, it could, I just thought having a file per Lambda made it very obvious where to find it. I feel like that `lambda-functions` file just keeps getting bloated over time. And in the `ingest` module we have a file per workflow task.

But if you want me to move it, I can